### PR TITLE
Jenkins get last buildnumber test

### PR DESCRIPTION
**Pull request by tudorlesan**

### CODE REVIEW

**Comment by tudorlesan** [comment]
not sure why the circleci failed, it worked locally.

File: content/io/cloudslang/jenkins/invoke_job.sl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        from jenkinsapi.jenkins import Jenkins
+        j = Jenkins(url, '', '')
+
+        job = j.get_job(job_name)
+        job.invoke()
+
+        return_code = '0'
+        result_message = 'Success'
+      except:
+        import sys
```
Comment (orius123) [inline content/io/cloudslang/jenkins/invoke_job.sl]: Why do you import sys?

Comment (orius123) [inline content/io/cloudslang/jenkins/invoke_job.sl]: Why do you import sys?

**Comment by orius123** [comment]
:+1: after fixing the comment fill free to merge.

File: content/io/cloudslang/jenkins/invoke_job.sl (hunk 1 above)
Comment (tudorlesan) [inline content/io/cloudslang/jenkins/invoke_job.sl]: I used it to print errors but not anymore and I forgot about it. I can delete it or I can put the error inside an error_message outputs. Whichever you preffer

**Comment by tudorlesan** [comment]
for some odd reason, after removing the unused import the test fails. looking into this..

**Comment by tudorlesan** [comment]
tested it locally multiple times and it works fine. not sure how I should proceed. can I access the execution.logs from circleci? I need more information that just the fact that an operation failed..

**Comment by tudorlesan** [comment]
it seems it is sporadic :-1:

**Comment by orius123** [comment]
maybe it's timeout issue.
you can find the builder.log under the artifacts section in circle-ci

**Comment by tudorlesan** [comment]
thanks

**Pull request merged**